plugins/rbac/src/utils/rbac-utils.ts

```diff
@@ -108,12 +108,20 @@ export const getMembersFromGroup = (group: GroupEntity): number => {
   return membersList || 0;
 };
 
-export const getPluginId = (
+export const getPluginInfo = (
   permissions: PermissionPolicy[],
-  permission: string | undefined,
-): string =>
-  permissions.find(p => p.policies?.find(pol => pol.permission === permission))
-    ?.pluginId || '-';
+  permissionName: string | undefined,
+): { pluginId: string; isResourced: boolean } => {
+  const permission = permissions.find(
+    p => p.policies?.find(pol => pol.permission === permissionName),
+  );
+  const pluginId = permission?.pluginId || '-';
+  const isResourced =
+    permission?.policies?.find(p => p.permission === permissionName)
+      ?.isResourced || false;
+
+  return { pluginId, isResourced };
```

Comment:
@debsmita1 I will suggest that you use reduce here instead of using find 3 times.